### PR TITLE
Regression: Remove self from fallback departments dropdown

### PR DESCRIPTION
**Pull request by KevLehman**

### CODE REVIEW

File: definition/IRoom.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export enum OmnichannelSourceType {
 	OTHER = 'other', // catch-all source type
 }
 
-export interface IOmnichannelRoom extends Omit<IRoom, 'default' | 'featured' | 'broadcast' | 'featured' | ''> {
+export interface IOmnichannelRoom extends Omit<IRoom, 'default' | 'featured' | 'broadcast' | ''> {
```
Comment (KevLehman) [inline definition/IRoom.ts:110]: This is an unrelated change to remove duplicated 'featured' from Omit.

File: client/components/RoomIcon/OmnichannelRoomIcon/OmnichannelCoreSourceRoomIcon.tsx [hunk 2]
```diff
@@ -27,5 +27,5 @@ export const OmnichannelCoreSourceRoomIcon = ({
 	size: ComponentProps<typeof Icon>['size'];
 }): ReactElement => {
 	const icon = iconMap[room.source?.type || 'other'] || 'headset';
-	return <Icon name={icon} size={size} color={colors[room.v.status || 'offline']} />;
+	return <Icon name={icon} size={size} color={colors[room.v?.status || 'offline']} />;
```
Comment (KevLehman) [inline client/components/RoomIcon/OmnichannelRoomIcon/OmnichannelCoreSourceRoomIcon.tsx:30]: Again, unrelated change, just for safety in the case a room doesn't have a `v` prop. This doesn't happen very often, but if it happens the UI will be unusable.

**Review by tiagoevanp — APPROVED** [review]
